### PR TITLE
Add 'metrics_reader.sh' for streaming workloads.

### PR DESCRIPTION
**Pull request by ConeyLiu**

### CODE REVIEW

File: bin/workloads/streaming/fixwindow/common/metrics_reader.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
+                      #!/bin/bash
```
Comment (carsonwang) [inline bin/workloads/streaming/fixwindow/common/metrics_reader.sh]: can you remove the extra spaces here?

File: bin/workloads/streaming/identity/common/metrics_reader.sh [hunk 2]
```diff
@@ -0,0 +1,36 @@
+                      #!/bin/bash
```
Comment (carsonwang) [inline bin/workloads/streaming/identity/common/metrics_reader.sh]: can you remove the extra spaces here?

File: bin/workloads/streaming/repartition/common/metrics_reader.sh [hunk 3]
```diff
@@ -0,0 +1,36 @@
+                      #!/bin/bash
```
Comment (carsonwang) [inline bin/workloads/streaming/repartition/common/metrics_reader.sh]: can you remove the extra spaces here?

File: bin/workloads/streaming/wordcount/common/metrics_reader.sh [hunk 4]
```diff
@@ -0,0 +1,36 @@
+                      #!/bin/bash
```
Comment (carsonwang) [inline bin/workloads/streaming/wordcount/common/metrics_reader.sh]: can you remove the extra spaces here?

File: conf/hibench.conf [hunk 5]
```diff
@@ -111,3 +112,13 @@ hibench.streambench.datagen.data1.name                  Seed
...
+
+#======================================================
+# MetricsReader for streaming benchmarks
+#======================================================
+# Number of sample records for `MetricsReader` (default: 5000000)
+hibench.streambench.metricsReader.sampleNum      5000000
+# Number of thread for `MetricsReader` (default: 20)
+hibench.streambench.metricsReader.threadNum      20
+# The dir where stored the report of benchmarks
+hibench.streambench.metricsReader.outputDir      /PATH/TO/DIR
```
Comment (carsonwang) [inline conf/hibench.conf]: can we set the default output dir to `${hibench.home}/report` ?

**Comment by ConeyLiu** [comment]
@carsonwang  thanks for the review, I've updated it.

**Comment by carsonwang** [comment]
Looks good. Thanks @ConeyLiu for the contribution!

**Pull request merged**